build.sbt

```
@@ -882,6 +899,20 @@ lazy val tests =
             Seq((Test / sourceDirectory).value / "scala-2.12+")
           case _ => Nil
         }
+      },
+      // temp hack
+      Test / unmanagedSourceDirectories ++= {
+        // unit-tests/native/src/test
+        val testDir = (Test / sourceDirectory).value
+        val rootTestDir =
+          testDir.getParentFile.getParentFile.getParentFile
+
+        val javaV = javaVersion.value
+        javaV match {
+          case v if (v >= 11) =>
+            Seq(rootTestDir / "require-jdk11" / "src" / "test")
+          case _ => Nil
+        }
```

We can make it more generic 
```suggestion
        (9 to (Global / javaVersion).value).map { v =>
          rootTestDir / s"require-jdk$v" / "src" / "test"
        }
```

---

I really like this suggestion. I haven't changed it yet as I am working on the source location.

---

Got this one - thanks.